frontend/package.json

```diff
@@ -27,7 +27,8 @@
     "react-hook-form": "7.45.4",
     "react-redux": "8.1.2",
     "react-router-dom": "6.15.0",
-    "react-toastify": "9.1.3"
+    "react-toastify": "9.1.3",
+    "vite-plugin-svgr": "3.2.0"
```

Comment:
vite-plugin-svgr should be a dev dependency

Reply:
done